doc/contrib/index.rst

```diff
@@ -34,6 +34,17 @@ repository and nearly all of its infrastructure.
 -  Make sure you have a `GitHub
    account <https://github.com/signup/free>`__.
 
+.. _contribindex-model:
+
+Model
+~~~~~
+
+Sharing actual science done with Arbor is the best contribution possible!
+
+- We encouraged the use of our Github Discussions board for public discussions on modelling: `Github Discussions <https://github.com/arbor-sim/arbor/discussions>`_.
+
+- Under the `arbor-contrib <https://github.com/arbor-contrib/>`_ organisation, some users have already shared their models. You can peruse these at your leisure, and of course add yours if you like to share! `Please contact us <https://docs.arbor-sim.org/en/stable/contrib/index.html#get-in-touch>`_ to have your model added to our list.
```

Comment:
the word organisation might be confusing: we know what a GitHub organisation is, however users might interpret it as an organisation that has to be joined.

How about:
```
* Users can share their models by adding them to the model collection on GitHub at `arbor-contrib <https://github.com/arbor-contrib/>`_.
```